src/Microsoft.ML.AutoML/Utils/StringEditDistance.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Microsoft.ML.AutoML.Utils
+{
+    public static class StringEditDistance
```

Comment:
Doesn't need to be public, as users won't need to access it directly.

Reply:
It's internal now.